### DISCUSSION TITLE
Support for other Charts from Recharts

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [mantinedev/mantine#6882] [Feature requests]
@rtivital 

Do you have a roadmap for supporting Recharts charts that you don't support yet?

e.g.
- https://recharts.org/en-US/api/FunnelChart
- https://recharts.org/en-US/api/SankeyChart
- https://recharts.org/en-US/api/RadialBarChart

I would like to contribute on any of them (e.g. Radial Barchart) if they are planned to be supported in a minor release of v7 and you have not yet worked on them at all and they are not annoying!

**Comment by rtivital** [comment]
Other charts are planned to be added over time in minor releases

**Reply by Kenzo-Wada** [reply]
Just Can't Wait!
Is there any chart that I can help you with?

**Reply by rtivital** [reply]
You can welcome to submit PRs with Funnel and Sankey charts

**Reply by Kenzo-Wada** [reply]
Thanx!
